azure-pipelines.yml

```diff
@@ -442,14 +442,47 @@ stages:
               vs2022_latest89:
                 PRODUCT: "SonarQube"
                 SQ_VERSION: "LATEST_RELEASE[8.9]"
-                SONAR_CFAMILYPLUGIN_VERSION: "6.20.5.49286"  # LATEST_RELEASE of CFAMILY is not compatible with old SQ
+                SONAR_CFAMILYPLUGIN_VERSION: "6.20.5.49286"  # LATEST_RELEASE of CFAMILY is not compatible with SQ 8.9
                 SONAR_CSHARPPLUGIN_VERSION: "DEV"
                 SONAR_VBNETPLUGIN_VERSION: "DEV"
                 MSBUILD_PATH: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2022\\BuildTools\\MSBuild\\Current\\Bin\\MSBuild.exe"
                 PLATFORMTOOLSET: "v140"
                 WINDOWSSDKTARGET: "10.0.17763.0"
                 JDKVERSION: "1.11"
                 TEST_SUITE: "**/SonarQubeTestSuite.java"
+              vs2017_latest99:
+                PRODUCT: "SonarQube"
+                SQ_VERSION: "LATEST_RELEASE[9.9]"
+                SONAR_CFAMILYPLUGIN_VERSION: "6.44.0.61773"  # LATEST_RELEASE of CFAMILY might not be compatible with SQ 9.9
+                SONAR_CSHARPPLUGIN_VERSION: "DEV"
+                SONAR_VBNETPLUGIN_VERSION: "DEV"
+                MSBUILD_PATH: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\MSBuild\\15.0\\Bin\\MSBuild.exe"
+                PLATFORMTOOLSET: "v140"
```

Comment:
Clean as you code: `PLATFORMTOOLSET` is constant for all matrix => should not be in matrix, but variables or environment instead.